dune/xt/common/test/float_cmp.cc

```diff
@@ -97,6 +97,39 @@ typename std::enable_if<XT::Common::is_matrix<T>::value, T>::type make_type(cons
     return Mat::create(VECSIZE, NUMCOLS, number);
 }
 
+template <class T,
```

Comment:
Just FYI these kinds of specializations can also be expressed, much more concisely IMO, via function overloads alone. See also https://github.com/dune-community/dune-gdt/blob/33dafdfb5a2615a374228c21fbde0f9074b20003/dune/gdt/spaces/cg/interface.hh#L139

Reply:
Ok thanks for your comment.